etcdctl/README.md

```diff
@@ -1118,6 +1118,14 @@ LOCK returns a zero exit code only if it is terminated by a signal and releases
 
 If LOCK is abnormally terminated or fails to contact the cluster to release the lock, the lock will remain held until the lease expires. Progress may be delayed by up to the default lease length of 60 seconds.
 
+#### How to grant permission to create locks to a role?
+￼
+The following command will allow locks to be created under  `/my_service_locks/*`:
+
+```bash
+etcdctl role grant-permission my_rw_locks --prefix=true readwrite _locks//my_service_locks/
```

Comment:
Could you help verify this? The "_locks" prefix looks strange to me. When we call `etcdctl lock some/path/to/lock_name`, the client is accessing key `some/path/to/lock_name/***`. So I don't see why there is an extra prefix of "_locks".

cc @hexfusion

Reply:
+1 to @jingyih I also don't understand how lock is relevant here. If I understood correctly `my_rw_locks` treated here as a name of role and `_locks//my_service_locks/` as key, here is how the role command looks like `role grant-permission [options] <role name> <permission type> <key> [endkey] [flags]` . Isn't this proposed command here similar to the examples already given here - https://github.com/etcd-io/etcd/blob/f1e2c22139461bcc218470cfa54b374db7181d5d/etcdctl/README.md#examples-16 ? I might be misunderstanding it, but in that case, it would be great if the command is documented here clearly with more details. Thanks!